tests/test_subworkflows.py

```diff
@@ -13,10 +11,10 @@
 import nf_core.modules
 import nf_core.subworkflows
 
-from .utils import GITLAB_URL, mock_api_calls
+from .utils import GITLAB_SUBWORKFLOWS_BRANCH, GITLAB_URL, mock_api_calls
 
 
-def create_modules_repo_dummy(tmp_dir):
+def create_subworkflows_repo_dummy(tmp_dir):
```

Comment:
I think that should be named `modules_repo` as it's a dummy `nf-core/modules` repo
```suggestion
def create_modules_repo_dummy(tmp_dir):
```